core/src/avm2/globals/flash/text/engine/TextBlock.as

```diff
@@ -48,83 +51,106 @@ package flash.text.engine {
-        
+
         public function get baselineFontDescription():FontDescription {
             return this._baselineFontDescription;
         }
-        
+
         public function set baselineFontDescription(value:FontDescription):void {
             this._baselineFontDescription = value;
         }
-        
+
         public function get baselineFontSize():Number {
             return this._baselineFontSize;
         }
-        
+
         public function set baselineFontSize(value:Number):void {
             this._baselineFontSize = value;
         }
-        
+
         public function get baselineZero():String {
             return this._baselineZero;
         }
-        
+
         public function set baselineZero(value:String):void {
             this._baselineZero = value;
         }
-        
+
         public function get bidiLevel():int {
             return this._bidiLevel;
         }
-        
+
         public function set bidiLevel(value:int):void {
             this._bidiLevel = value;
         }
-        
+
         public function get lineRotation():String {
             return this._lineRotation;
         }
-        
+
         public function set lineRotation(value:String):void {
             if (value == null) {
                 throw new TypeError("Error #2007: Parameter lineRotation must be non-null.", 2007);
             }
             // TODO: This should validate that `value` is a member of TextRotation
             this._lineRotation = value;
         }
-        
+
         // Note: FP returns a copy of the Vector passed to it, so modifying the returned Vector doesn't affect the actual internal representation
         public function get tabStops():Vector.<TabStop> {
             return this._tabStops;
         }
-        
+
         // Note: FP makes a copy of the Vector passed to it, then sets its internal representation to that
         public function set tabStops(value:Vector.<TabStop>):void {
             this._tabStops = value;
         }
-        
+
         public function get textJustifier():TextJustifier {
             return this._textJustifier;
         }
-        
+
         public function set textJustifier(value:TextJustifier):void {
             this._textJustifier = value;
         }
-        
+
         public function get content():ContentElement {
             return this._content;
         }
-        
+
         public function set content(value:ContentElement):void {
             this._content = value;
         }
+
+        public native function createTextLine(previousLine:TextLine = null, width:Number = 1000000, lineOffset:Number = 0, fitSomething:Boolean = false):TextLine;
+
+        public function get textLineCreationResult():String {
+            return this._textLineCreationResult;
+        }
+
+        public function get firstLine():TextLine {
+            return this._firstLine;
+        }
+
+        public function get lastLine():TextLine {
+            return this._firstLine;
```

Comment:
is this a copy error, or an intentional simplification? If the latter, can you add a comment, and maybe the stub marker?